.gitlab-ci.yml

```diff
@@ -5,19 +5,22 @@ include:
 
 variables:
   SERVICE_PORT: 80
+  DOCKER_BUILD_ARG_REACT_APP_SENTRY_DSN: '$GL_SENTRY_DSN'
+  SASS_PATH: 'node_modules:src/assets/styles'
```

Comment:
Should `SASS_PATH` be passes similarly to other variables in `.env`, i.e. `DOCKER_BUILD_ARG_SASS_PATH: ...` and just `ARG SASS_PATH` in the dockerfile?

Reply:
Corrected variable naming. I have not been able to successfully run `docker build .` locally with just `ARG SASS_PATH` instead of giving a default `ARG SASS_PATH="node_modules:src/assets/styles"`

Reply:
That's probably due to `.env` now being in dockerignore. We should still pass these variables to docker-compose, by for example adding a separate file and `env_file: docker-compose.env.yaml` to `docker-compose.yml`.